lms/djangoapps/verify_student/houston_encrypt.py

```diff
@@ -0,0 +1,125 @@
+
+import json
+import logging
+import requests
+import boto3
+
+from datetime import datetime, timedelta
+from email.utils import formatdate
+from django.conf import settings
+from django.core.urlresolvers import reverse
+log = logging.getLogger(__name__)
+import pdb
+
+from lms.djangoapps.verify_student.ssencrypt import (
+    generate_signed_message
+)
+
+def create_houston_request(receipt_id, photo_id_key, copy_id_photo_from=None):
+    """
+    Construct the HTTP request to the photo verification service.
+    Keyword Arguments:
+        copy_id_photo_from (SoftwareSecurePhotoVerification): If provided, re-send the ID photo
```

Comment:
if this is for houston, why SoftwareSecurePhotoVerification?

Reply:
Because it uses the SoftwareSecureVerification model and it's fields. This file contains two extra functions which extend the model and we create a "houston request" (post to Houston Stu) but everything else stays the same.